speedcord/http.py

```diff
@@ -47,6 +58,13 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
 class HttpClient:
     def __init__(self, token, *, baseuri="https://discord.com/api/v7", loop=asyncio.get_event_loop()):
+        """
+        Created by speedcord.Client when started. Runs requests sent to it by creating Route 
+        objects and also handles rate limits with LockManager. 
```

Comment:
The lockmanager isn't related to ratelimits. Its simply a extension to the default lock context manager but with a function to prevent it from unlocking

Reply:
I guess I misunderstood it then - I thought that you prevent it from unlocking due to ratelimits.